### PR TITLE
Feat/tile loader no esm

### PR DESCRIPTION
**Pull request by zachferland**
Same as https://github.com/ceramicstudio/js-3id-did-provider/pull/144 but remove esm commit

### CODE REVIEW

**Comment by oed** [comment]
@zachferland can you provide some more details please? Should this PR be used or the other one? What's the reason for separating the two? Quite hard to follow what's going on here!

**Comment by zachferland** [comment]
Changed to draft, but main branch already includes esm changes which are breaking and i assuming will be timed with the rest of the ceramic release (esm), this includes same as other PR without breaking, allowing minor release to be consumed by 3id-connect before upgrade

**Comment by oed** [comment]
Afaik js-ceramic doesn't depend on js-3id-did-provider, so there should be no problem with releasing this as ESM now?

**Comment by zachferland** [comment]
yeah but its harder to consume both module types at same time, and i dont know the whole plan here, seemed easier for consumers to have similar breaking changes together ( i know it is in 3id-connect at least)

**Comment by oed** [comment]
I thought js-glaze fully ESM already? Consumers already have to deal with this.

**Comment by oed** [comment]
Any input on this @PaulLeCam? I don't have strong feelings either way.

**Comment by PaulLeCam** [comment]
@oed we discussed this a bit with @zachferland, I think it makes sense to continue being able to make non-breaking releases before we're fully ESM in the stack.
We'll need to have an ESM release of Ceramic so we can really start updating Glaze packages to the new setup and then the 3ID packages.

**Comment by oed** [comment]
Ok, sounds good 👍
